pages/id/docs/guides/nodejs-docker-webapp.md

```diff
@@ -66,10 +66,10 @@ touch Dockerfile
 
 Buka `Dockerfile` di editor teks favorit Anda
 
-Hal pertama yang perlu kita lakukan adalah menentukan dari image apa kita ingin membangun. Di sini kita akan menggunakan LTS (dukungan jangka panjang) versi `16` terbaru dari `node` tersedia dari [Docker Hub](https://hub.docker.com/_/node):
+Hal pertama yang perlu kita lakukan adalah menentukan dari gambar apa kita ingin membangun. Di sini kita akan menggunakan versi terbaru LTS (dukungan jangka panjang) `18` dari `node` tersedia dari [Docker Hub](https://hub.docker.com/_/node):
 
 ```docker
-FROM node:16
+DARI node: 18
```

Comment:
This seems wrong... lol

Reply:
> This seems wrong... lol `FROM` is a keyword...

Initially, I also thought the same way, but Crowdin does not show it as a keyword. 👀
Thank you for the review, I'll fix it :)

Reply:
@ovflowd 👋

you could look into crowdin's settings to disable code translation (this is what's been done on webdriver.io). Unless you don't have access to .

Reply:
I don't see any option for "disabling code translations", but will give an eye :)

Reply:
I was able to find the setting and disabled the ability to translate code blocks on both pure Markdown and MDX. I also disabled "content segmentation" or Markdown and I'm curious to see if this affects positively these MDX issues we've having?

Reply:
> Initially, I also thought the same way, but Crowdin does not show it as a keyword. 👀 Thank you for the review, I'll fix it :)

As for @HinataKah0 's pointing out：Just a suggestion：We tend to keep what the comments and commands of codes what they really are (Not translations), bcoz we will execute these commands step by step :)